pkg/apiserver/api_server.go

```diff
@@ -211,6 +265,53 @@ func (s *ApiServer) updatePipe(c *gin.Context) {
 	c.JSON(http.StatusOK, gin.H{"msg": "update success"})
 }
 
+func (s *ApiServer) pausePipe(c *gin.Context) {
+	name := c.Param("name")
+
+	err := s.updatePauseSpecWithRetry(name, true)
+	if err != nil {
+		pauseErrorMsgCount.WithLabelValues(name).Add(1)
```

Comment:
I think we just need a general `api error count` with endpoint(path/method) as label. no need to make one metric for each kind of error.

Reply:
My consideration is that `pause` and `resume` generally should never fail. For pipeline that does not exist, it does not return error. For pipeline that exists it should not have error.

For other API errors, it may happen very often since human trigger the api very often, so no need to track those errors, and it's not easy to distinguish human error or internal cluster error.